ui/packages/catalog-ui-search/src/main/webapp/component/select/select.collection.view.js

```diff
@@ -161,7 +161,7 @@ define([
                     }).length > 1))) {
                 return false;
             }
-            if (this.getAppropriateString(child.get('value')).indexOf(filterValue) === -1) {
+            if (this.getAppropriateString(child.get('label')).indexOf(filterValue) === -1) {
```

Comment:
We could use both `value` and `label` for underlying autocomplete and get suggestions from both.

Reply:
We discussed in our sprint that it should show what is configured in the admin ui.

Reply:
I think that in general, using value was confusing for users because it doesn't match the configured alias label (which a normal user cannot see).  I'm also not sure that a normal user will know the underlying metacard attribute name, so if the alias is nowhere near the actual attribute name the suggestion makes no sense.

Reply:
I think this is a good topic for discussion

Reply:
I would say only use the label, using the value would be confusing to me.

Reply:
I'm not sure if label is guaranteed though, might need to fall back to value for undefined labels.